app/DynamicsAdapter/Fams3Adapter.Dynamics/SearchRequest/SearchRequestService.cs

```diff
@@ -35,5 +37,14 @@ public async Task<SSG_Identifier> UploadIdentifier(SSG_Identifier identifier, Ca
             return await this._oDataClient.For<SSG_Identifier>().Set(identifier).InsertEntryAsync(cancellationToken);
         }
 
+        public async Task<SSG_Address> UploadAddress(SSG_Address address, CancellationToken cancellationToken)
```

Comment:
can we just change all the upload to create ? I think I wrongly used that word in the standups ;-)

Reply:
I like upload, it is more clear. But ok, I will change to "create".